ecl.h

```diff
@@ -47,18 +47,14 @@
 #define ecl_elapsed_time hrt_elapsed_time
 using ecl_abstime = hrt_abstime;
 
-#define ECL_INFO PX4_INFO
-#define ECL_WARN PX4_WARN
-#define ECL_ERR	 PX4_ERR
-
-#if defined(__PX4_POSIX)
-#define ECL_INFO_TIMESTAMPED(X) PX4_INFO("%llu: " X, (unsigned long long)_imu_sample_delayed.time_us)
-#define ECL_WARN_TIMESTAMPED(X) PX4_WARN("%llu: " X, (unsigned long long)_imu_sample_delayed.time_us)
-#define ECL_ERR_TIMESTAMPED(X) PX4_ERR("%llu: " X, (unsigned long long)_imu_sample_delayed.time_us)
+#if defined(__PX4_NUTTX)
+#  define ECL_INFO PX4_DEBUG
```

Comment:
I'd still like to figure out a way to the SITL experience to be as close to identical as possible, but let's give this a try.